samtranslator/policy_templates_data/policy_templates.json

```diff
@@ -117,7 +117,8 @@
               "dynamodb:UpdateItem",
               "dynamodb:BatchWriteItem",
               "dynamodb:BatchGetItem",
-              "dynamodb:DescribeTable"
+              "dynamodb:DescribeTable",
+              "dynamodb:ConditionCheckItem"
```

Comment:
verified this policy from [here](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/transaction-apis-iam.html)